UEFITool/uefitool.pro

```diff
@@ -9,6 +9,7 @@ DEFINES += "U_ENABLE_ME_PARSING_SUPPORT"
 DEFINES += "U_ENABLE_GUID_DATABASE_SUPPORT"
 
 HEADERS += uefitool.h \
+ ../common/uinttypes.h \
```

Comment:
Move common file to a common group please. E.g. near `../common/ubytearray.h`.

Reply:
fixed in 582910fb96edefffa7c51442cc39681f90ece010